index.js

```diff
@@ -88,7 +88,7 @@ JSHinter.prototype.testGenerator = function(relativePath, passed, errors) {
     errors = ""
   }
 
-  return "module('JSHint - " + path.dirname(relativePath) + "');\n" +
+  return "QUnit.module('JSHint - " + path.dirname(relativePath) + "');\n" +
          "test('" + relativePath + " should pass jshint', function() { \n" +
```

Comment:
Can you also:

* change this to `QUnit.test`
* pass in `assert` as an argument to the test callback function
* use `assert.ok` instead of the global `ok`